nss-day-cohort-45/tabloidfullstack-gibbering-gizmos
# Edit Post

## Changes:

Backend:

- Added backend logic for updating a post in the PostRepository
- Added logic for the API endpoint in the PostController

Frontend:

- Added PostEdit.js to handle editing a post with logic to make sure only the owner can edit
- Updated ApplicationViews to reflect the new edit post route
- Refactored Post.js and PostDetails.js to show edit/delete buttons when the logged-in user owns the post
- Post components now have their own directory

## To Test:

1. Run the API and Client
2. Make sure you can see edit/delete buttons for posts the logged-in user owns
3. Click the edit button in the post list or "My Feed"
4. Make changes and save the post
5. Note that once edited it will be marked as NOT approved so will not show up on the post list (check your feed instead)
